en-GB/Term 2/Brain Game/Brain Game.md

```diff
@@ -216,7 +216,7 @@ Instead of your character just saying `yes! :)` or `nope :(` to the player, lets
 ## Save your project { .save }
 
 ## Challenge: Improved animation {.challenge}
-Can you improve your graphics animation? You could code the tick and cross so that they fade out as well as fade in. Or, you could use other cool effects:
+Can you improve your graphic's animation? You could code the tick and cross so that they fade out as well as fade in. Or, you could use other cool effects:
```

Comment:
I think this is either graphics, or graphics', but not graphic's.
I suspect we could avoid this entirely by reworking as "Can you improve the animation of your graphics?"